src/core/agent/services/connectionService.ts

```diff
@@ -132,6 +132,9 @@ class ConnectionService extends AgentService {
         };
       }
     }
+    await this.props.signifyClient.contacts().update(connectionId, {
```

Comment:
The OOBI resolution is async - so we should be calling this for when the long running operation completes for the OOBI.

We have some logic to wait 2 seconds if a connection has complete - we should get rid of that and simplify. For normal connections, always track the long running operation. When it completes, create the contact and delete the long running operation record.